x-pack/plugins/security_solution/cypress/tasks/alerts.ts

```diff
@@ -125,6 +128,16 @@ export const openAlerts = () => {
   cy.get(OPEN_ALERT_BTN).click();
 };
 
+export const selectCountTable = () => {
+  cy.get(CHART_SELECT).click({ force: true });
```

Comment:
Q: What does `{ force: true }` do and why do we need it in this context?

Reply:
The `force` parameter to Cypress actions [bypasses](https://docs.cypress.io/guides/core-concepts/interacting-with-elements#Forcing) some of the checks that might otherwise prevent Cypress from clicking on an element.

In `main`, the Cypress tests in this file, [alerts.ts](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/cypress/tasks/alerts.ts):

- Utilize `click({ force: true })` a total of `11` times throughout the file
- Also utilize `click()` (without `force`) a total of `10` times throughout the file

So it's ~ a 50 / 50 split in the existing code in [alerts.ts](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/cypress/tasks/alerts.ts), but many functions in that file also mix the use `{ force: true }` for the first invocation of `click` in a function, but NOT for the second `click`:

Example 1:

```ts
export const addExceptionFromFirstAlert = () => {
  cy.get(TIMELINE_CONTEXT_MENU_BTN).first().click({ force: true });
  cy.get(ADD_EXCEPTION_BTN).click();
};
```

Example 2:

```ts
export const openFirstAlert = () => {
  cy.get(TIMELINE_CONTEXT_MENU_BTN).first().click({ force: true });
  cy.get(OPEN_ALERT_BTN).click();
};
```

Example 3:

```ts
export const openAlerts = () => {
  cy.get(TAKE_ACTION_POPOVER_BTN).click({ force: true });
  cy.get(OPEN_ALERT_BTN).click();
};
```

With the aim of reducing flakey test runs, the new functions follow the established convention above and use `{ force: true }` for the first `click` in a function.